example/reader_util.c

```diff
@@ -605,6 +605,7 @@ void ndpi_flow_info_free_data(struct ndpi_flow_info *flow) {
 #endif
 
   if(flow->tcp_fingerprint) ndpi_free(flow->tcp_fingerprint);
+  if(flow->tcp_fingerprint_raw) ndpi_free(flow->tcp_fingerprint_raw);
```

Comment:
If your goal is only to export the raw fingerprint to the json file, this change is useless. Please, remove it